src/ferui-components/tree-view/interfaces.ts

```diff
@@ -0,0 +1,108 @@
+import { TemplateRef } from '@angular/core';
+
+/**
+ * Tree Node Data Interface
+ */
+export interface TreeNodeData<T> {
+  data: T;
+  label: string;
```

Comment:
Can we replace `label` by `field` please ? Because for me, a label is something that you display on-screen but there, you're referring to a field within the data object.

Reply:
This was intended to represent something that is directly displayed on the screen vs. a field of the data object. Not sure if that is how it was implemented or not @ariquinones ?

Reply:
Updated

Reply:
why? Seems like it would be easier on everyone if you just made the developer provide the label directly